.github/workflows/test.yml

```diff
@@ -143,16 +143,16 @@ jobs:
       - name: Run main tests
         if: matrix.test == 'main'
         # running the "main" tests means "all tests that aren't auth"
-        run: pytest -m "not auth" -vx --cov binderhub
+        run: pytest -m "not auth" -vx --cov binderhub --durations=0
```

Comment:
I think showing the durations for all tests creates very large logs. How about the top 10? This is also the value we use in repo2docker. (same for all three instances of `--durations`